base/docs/helpdb/Dates.jl

```diff
@@ -49,11 +50,55 @@ All characters not listed above are treated as delimiters between date and time
 Dates.DateTime(dt::AbstractString, format::AbstractString)
 
 """
+    format(dt::TimeType, format::AbstractString; locale="english") -> AbstractString
+
+Construct a string by using a `TimeType` and applying the pattern provided by the
```

Comment:
And here "`TimeType` object" would be better, as the name can be confusing (could have been better called `AbstractTime` or something like that).